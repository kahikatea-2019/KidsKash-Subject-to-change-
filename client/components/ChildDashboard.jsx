import React from 'react'
import MaterialDashboard from './MaterialDashboard'
// import { connect } from 'react-redux'

import WishList from './WishList'
import StarBank from './StarBank'
import WishInput from './WishInput';

export default function ChildDashboard () {
  const child = {
    id: 1,
    child_name: 'Amy',
    display_url: 'https://i.pinimg.com/736x/62/c5/c0/62c5c004441b4bd92e9228198aa17aeb.jpg',
    parents_id: '1'
  }

  return (
    <React.Fragment>
      <div className="cd-body">
        <div className="high-frame">
          <div className="welcome-user">
            {`Hello, ${child.child_name}!`}
          </div>
        </div>
        <div className="low-frame">
          <WishInput/>
          <WishList id={child.id}/>
          <StarBank id={child.id}/>
        </div>

        <MaterialDashboard/>

      </div>
    </React.Fragment>
  )
}
