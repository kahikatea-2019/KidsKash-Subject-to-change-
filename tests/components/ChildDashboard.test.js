import React from 'react'
import { shallow } from 'enzyme'
import ChildDashboard from '../../client/components/ChildDashboard'

