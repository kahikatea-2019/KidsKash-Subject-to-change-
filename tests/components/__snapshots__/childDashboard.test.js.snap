// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `ShallowWrapper {}`;

exports[`ChildDashboard renders correctly 1`] = `ShallowWrapper {}`;
